### PR TITLE
Introduce an IMAP simulation forked from gatilng-imap project

### PR DESCRIPTION
**Pull request by aduprat**
To be reviewed: the last commit: fbd5d9c

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -15,14 +30,4 @@ enablePlugins(GatlingPlugin)
...
-libraryDependencies += "io.gatling.highcharts" % "gatling-charts-highcharts" % gatlingVersion
-
-libraryDependencies += "com.github.azakordonets" %% "fabricator" % "2.1.5"
-
-// Dependencies for local Courier library
-libraryDependencies += "com.sun.mail" % "javax.mail" % "1.6.2"
-libraryDependencies += "javax.activation" % "activation" % "1.1.1"
-libraryDependencies += "org.bouncycastle" % "bcpkix-jdk15on" % "1.60" % Optional
-libraryDependencies += "org.bouncycastle" % "bcmail-jdk15on" % "1.60" % Optional
+lazy val gatlingImap = ProjectRef(uri("git://github.com/linagora/gatling-imap.git"), "gatling-imap")
```
Comment (rouazana) [inline build.sbt]: why not, but we could plan to put in on maven central

**Review by rouazana — APPROVED** [review]
no PR on gatling-imap to remove the exclusion?

File: src/main/scala-2.12/org/apache/james/gatling/imap/scenari/ImapFullScenario.scala [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  val heavyUser = repeat(3)(receiveEmail)
+    .repeat(2)(readLastEmail)
+  val lightUser = receiveEmail
+    .exec(readLastEmail)
+
+  def generate(duration: Duration, users: Seq[Future[User]]): ScenarioBuilder =
+    scenario("Imap")
+      .feed(UserFeeder.createCompletedUserFeederWithInboxAndOutbox(users))
+      .pause(1.second)
+      .during(duration) {
```
Comment (blackheaven) [inline src/main/scala-2.12/org/apache/james/gatling/imap/scenari/ImapFullScenario.scala:40]: I feel it a bit weird to have a `during()` here and `forever()` later, what about nothing on top and `duration()`s later?

Comment (mbaechler) [reply src/main/scala-2.12/org/apache/james/gatling/imap/scenari/ImapFullScenario.scala:40]: That's not the topic of that ticket (even if you comment makes sense)

Comment (blackheaven) [reply src/main/scala-2.12/org/apache/james/gatling/imap/scenari/ImapFullScenario.scala:40]: Oh, I thought it was new code, not imported code

**Review by blackheaven — APPROVED** [review]

**Review by trantienduchn — APPROVED** [review]

**Comment by aduprat** [comment]
`push -f` in order to solve conflict

**Review by blackheaven — APPROVED** [review]

**Review by remk — APPROVED** [review]

**Pull request merged**